integration-tests/src/test/scala/chiselTests/util/experimental/minimizer/MinimizerSpec.scala

```diff
@@ -272,4 +272,39 @@ trait MinimizerSpec extends AnyFlatSpec with ChiselScalatestTester with Formal {
       BitPat(s"b${Seq(N, X, X, X, X, X, X, X, X, A2_X, A1_X, IMM_X, DW_X, FN_X, N, M_X, X, X, X, X, X, X, X, CSR_X, X, X, X, X).reduce(_ + _)}")
     ))
   }
+
+  "Bug from RC FPU" should "pass" in {
+    minimizerTest(TruthTable.fromString(
+      """?????00??????????????????1000011->0
+        |?????00??????????????????1000111->0
+        |?????00??????????????????1001011->0
+        |?????00??????????????????1001111->0
+        |0000000??????????????????1010011->0
+        |?????????????????010?????0000111->?
+        |?????????????????010?????0100111->0
+        |0000100??????????????????1010011->0
+        |0001000??????????????????1010011->0
+        |0001100??????????????????1010011->0
+        |0010000??????????000?????1010011->0
+        |0010000??????????001?????1010011->0
+        |0010000??????????010?????1010011->0
+        |0010100??????????000?????1010011->0
+        |0010100??????????001?????1010011->0
+        |010110000000?????????????1010011->0
+        |1010000??????????000?????1010011->0
+        |1010000??????????001?????1010011->0
+        |1010000??????????010?????1010011->0
+        |110000000000?????????????1010011->0
+        |110000000001?????????????1010011->0
+        |110000000010?????????????1010011->0
+        |110000000011?????????????1010011->0
+        |110100000000?????????????1010011->0
+        |110100000001?????????????1010011->0
+        |110100000010?????????????1010011->0
+        |110100000011?????????????1010011->0
+        |111000000000?????000?????1010011->0
+        |111000000000?????001?????1010011->0
+        |111100000000?????000?????1010011->0
+        |                                  ?""".stripMargin))
```

Comment:
From this table(copy from RC). it semantically means just return 0 is enough.
However Decoder didn't cover this.